### ISSUE TITLE
[Legitimate Site Blocked] etcerscan.com

### ISSUE THREAD

**Issue by HebeBlock** [MetaMask/eth-phishing-detect#9425]
https://etcerscan.com/

Hi, we built etcerscan.com for EthereumClassic, it looks like the site is blocked by mistake

<img width="1451" alt="image" src="https://user-images.githubusercontent.com/39998387/201934678-17080cca-5546-40ff-855f-8f9d21bdfe75.png">
<img width="1442" alt="image" src="https://user-images.githubusercontent.com/39998387/201934717-98eb382d-ec56-4d62-a965-cad83554c4ae.png">

**Comment by HebeBlock** [comment]
please check us

https://twitter.com/BlockHebe/status/1592498982697066496

**Comment by HebeBlock** [comment]
@409H

**Comment by Tweaker177** [comment]
I can personally vouch for the validity of said developer / dev team as I personally got to know them when they first started Hebeswap, excited to have them building on ETC but cautiously optimistic as a long time investor and influencer for ETC, we had been let down in the past.  

After speaking with them, visiting their GitHub(s), getting their iOS app, I started using their dapps, participating in the liquidity pools, bought 5 HENS addresses in total, several for MetaMask wallets that would be great if MetaMask visited the HENS GitHub and implemented the API so all the .etc domains / wallets can function and be displayed as intended in MetaMask. (I believe it’s only a few lines of code that needed added on your end).


Anyways, I’ve been watching the progress Hebe team has made on the ETC Explorer via Twitter (the same account posted that has posted every other Hebe release).  They are legit. The Hebe developers are the heroes ETCARMY had been waiting on, all their projects are open source and am  pretty sure also verified in Solidity’s Sourcify.    They also work, I’ve tested and am a user of most of their services including the ETC Chrome browser seb3 portal and wallet. To my knowledge they haven’t had a single security breach, and I’ve never seen any outages with them, unlike Solana, and by extension everything built on it.

Perhaps someone who does not like Ethereum Classic or just isn’t familiar with it assumed because of the similar name, it’s fake, but I don’t believe that to be the case any more than I believed every time the coin went up in value it was from people buying it by mistake.  Billions in market cap are not mistakes.

ETCERScan was obviously not fake ETHer-scan, as it has data and advertisements very easily distinguishable as being part of the Ethereum Classic ecosystem.  

Thanks, and if you can plz implement the HENS API into MetaMask that would be awesome!

**Comment by HebeBlock** [comment]
HI, it's been 2 days, can anyone help?

**Comment by HebeBlock** [comment]
HI, it's been 3 days, can anyone help?

**Comment by shibcmemes** [comment]
Hi @deshvin 

You added shibaclassic.sc to the allowlist https://github.com/MetaMask/eth-phishing-detect/issues/9109 here.

I have been using HebeBlock contracts and solutions/infra for the past 3 months, consistently, daily.

They are very good for the ETC ecosystem and cryptocurrency as a whole, and I ask for our previous interactions, and the effort you have seen I made for my own due diligence to clear shibaclassic.sc (I am just a volunteer meme account who likes the token).

If you need a run down of their project, I would gladly do so off the top of my head and heart - again, these guys are fundamental to ETC/space, and we do need this infrastructure in order to function, having the exploder blocked is counterproductive.

Thank you a lot.

**Comment by HebeBlock** [comment]
HI, it's been 5 days, can anyone help?
We are still waiting for your reply

**Comment by shibcmemes** [comment]
Hi again @deshvin @AlexHerman1 - this is the most functioning and feature rich block explorer we have, it's incredibly important.

Again, this is really crippling production and productivity, having to cycle between each project in our ecosystem being filtered/blocked. We do appreciate your help with Shiba Classic - Malwarebytes had us whitelisted in 15 mins, from 2 volunteers, too.

This is critical public infra, not just related to ETC. Can we please get this sorted ASAP.

**Comment by shibcmemes** [comment]
@deshvin @AlexHerman1 please?

**Comment by Tweaker177** [comment]
I know for a fact that MetaMask is building during this unprecedented bear market.  Bear markets are the best time for building.  Yet you guys continue to ignore this legitimate issue.

For real, shame on you.  You should sell the knockoff version of ETC (I think they call it ETH) and start supporting the real thing.  Your blind support for a contentious fork with no values that align with being a de decentralized blockchain, while discriminating against the number one proof of work smart contracts chain (ECT) is flat out appalling.   It's your refusal to correct false information and assumptions that keeps so many people from the truth.   

How much is Vitalek paying you to do ETC such a big denial of service?

**Comment by Julucha** [comment]
At first glance it looks extremely similar to fake marketplace sites that
are used as wallet drainers. Again, my apologies; the domain can be removed
from the blocklist.

—

Le jeu. 5 janv. 2023 à 04:43, i0s_tweak3r ***@***.***> a
écrit :

**Comment by Tweaker177** [comment]
At first glance it looks like the closed source Etherscan feels threatened that it’s EaaS (Explorer as a Service) business could become completely unnecessary as DECENTRALIZED and OPEN SOURCE solutions like etcerscan are built, removing the need to spend money on what could and should be a free public resource.   

Let’s not pretend like this was an “honest mistake.”  It took you over two months to even respond to an “at a glance” banning of a block explorer for one of the biggest chains by market cap, and the oldest blockchain besides Bitcoin.  

At $2.5 Billion dollars of market cap, (which is extremely undervalued) your free decision to not do the job you were hired and are paid to do caused a lot of harm to everyday MetaMask users, confused about why things weren’t working. Also, you cost the Hebe team credibility, by making it appear their block explorer didn’t work, or even worse, that it was a con to be used as a phishing site;  that kind of damage you can’t  just buy back. 

You sure do owe a lot to the Hebe team, though, and ETC users in general.  An official apology would be a good place to start. Integrating the HENS API (which had you already done, there would never have been any confusion about the etcerscan site) would be something quick, easy and painless.  

Please consider doing what is right for your users going forward rather then what helps you keep an outdated narrative alive while propagating more lies and misleading info regarding Ethereum Classic.  

Also, in case you really are that ignorant, stop by Ethereumclassic.org and get educated.  It’s been updated since 2017- so should your knowledge base of the chain and it’s ecosystem.
